app/Resources/views/category/detail.html.twig

```diff
@@ -4,5 +4,6 @@
     <div class="row">
         {% include 'components/list.html.twig' %}
     </div>
+    {% include ":components:paginator.html.twig" %}
```

Comment:
Prečo je tu použitý zápis **:**components**:**paginator.html.twig (s dvojbodkami) a vyššie je normálne components**/**list.html.twig (s lomítkom)? Je to iba vec zápisu, alebo to má aj iný význam, príp. sú medzi týmito zápismi aj funkčné rozdiely?